software/Test_Robot_Software/follow_a_ball.py

```diff
@@ -0,0 +1,100 @@
+import image_processor
```

Comment:
All different logical behaviors of the robot should be unified in a state machine or similar. This makes modifying and adding functionality possible. Separate fragments are useless in the context of the competition.